cli/args/flags.rs

```diff
@@ -6450,6 +6470,15 @@ mod tests {
       }
     );
 
+    let r = flags_from_vec(svec!["deno", "init", "--help"]);
+    assert_eq!(
+      r.unwrap(),
+      Flags {
+        subcommand: DenoSubcommand::Init(InitFlags { dir: None }),
+        ..Flags::default()
+      }
+    );
```

Comment:
I think this test will fail. Once it's reverted then the PR looks good.

Reply:
I am sorry, that's my testing test for help. Please check once again @dsherret